src/credentials/FabricTable.cpp

```diff
@@ -554,6 +554,24 @@ CHIP_ERROR FabricTable::FetchRootCert(FabricIndex fabricIndex, MutableByteSpan &
     return mOpCertStore->GetCertificate(fabricIndex, CertChainElement::kRcac, outCert);
 }
 
+CHIP_ERROR FabricTable::FetchPendingNonFabricAssociatedRootCert(MutableByteSpan & outCert) const
```

Comment:
Please add a unit test to fabrictable for this method

Reply:
Good point; I was trying to figure out how to test this in YAML, but unit test is easy.  https://github.com/project-chip/connectedhomeip/pull/21016/commits/dbc9858be719b480895a757d8214a03b56adfb68